modules/ossm-mirror-repository.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/preparing-ossm-installation.adoc
+
+[id="ossm-mirror-repository_{context}"]
+= Mirroring the {product-title} image repository
+
+Mirror the {product-title} image repository to use during cluster installation
+or upgrade.
+
+.Prerequisites
+
+* You configured a mirror registry to use in your restricted network and
```

Comment:
This would be a good place to insert the link to the OCP docs.  "Configured a mirror registry as described in [LINK]"

Reply:
moved this to a different file